rbac/api/models.py

```diff
@@ -94,3 +94,20 @@ def username(self, value: Optional[str]) -> None:
     bearer_token: str = ""
     client_id: str = ""
     is_service_account: bool = False
+
+
+class FilterQuerySet(models.QuerySet):
+    """Queryset for filtering."""
+
+    _public_tenant = None
+
+    @classmethod
+    def _get_public_tenant(cls) -> Tenant:
+        """Get or set public tenant."""
+        if cls._public_tenant is None:
+            cls._public_tenant = Tenant.objects.get(tenant_name="public")
+        return cls._public_tenant
+
+    def non_custom_only(self):
```

Comment:
What about renaming 
-  `with_public_tenant` 
- `public_tenant_only`

(I assuming that we are going to remove system flag. )